doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1690,6 +1696,8 @@ Response code:
           section for details.
           + values - when used, the response payload uses `values` simplified entity representation. See corresponding
           section for details.
+          + unique - when used, the response payload uses `values` simplified entity representation. And, recurring values are left out.
+          See corresponding section for details.
```

Comment:
greping in current .apib shows 4 occurrences of "+ values". However, this PR only includes three additions (typos appart) from "+ unique". Maybe one piece of documentation is missing yet?